Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -642,6 +657,16 @@ for (const auto & particle_diag : particle_diags) {
         pc->getCharge(), pc->getMass(),
         isBTD, isLastBTDFlush);
     }
+
+    auto hasOption=m_OpenPMDoptions.find("FlattenSteps");
+    bool flattenSteps = isBTD && (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
```

Comment:
```suggestion
    const bool flattenSteps = isBTD && (m_Series->backend() == "ADIOS2") && (hasOption != std::string::npos);
```
Could you please declare this as a `const` ?